ivy/stateful/norms.py

```diff
@@ -192,6 +194,8 @@ def _forward(
         ret
             The outputs following the batch normalization operation.
         """
```

Comment:
I agree with ved, I'm not seeing why this logic is necessary. training mode is now dictated by `self.training` so surely no need for `training` to also be an argument to the forward, and we just pass `self.training` to the functional batch norm directly?

Reply:
agreed, we can get rid of the argument from forward entirely.